openwisp_radius/settings.py

```diff
@@ -81,6 +81,7 @@ def get_default_password_reset_url(urls):
 PASSWORD_RESET_URLS.update(get_settings_value('PASSWORD_RESET_URLS', {}))
 DEFAULT_PASSWORD_RESET_URL = get_default_password_reset_url(PASSWORD_RESET_URLS)
 SMS_VERIFICATION_ENABLED = get_settings_value('SMS_VERIFICATION_ENABLED', False)
+COA_ENABLED = get_settings_value('COA_ENABLED', False)
```

Comment:
Please start adding this to the settings page and add a draft page for coa which is linked from this setting.